packages/eyes/src/index.ts

```diff
@@ -58,6 +81,20 @@ async function runTests(url: string) {
   }
 }
 
+function logEyesResult({name, status, isNew, appUrls}: any) {
+  const isError = status === 'Unresolved' && isNew.toString() === 'false';
```

Comment:
Isn't isNew a boolean? I think isError should be status !== 'Passed' instead of status === 'Unresolved'.